lib/auth/nextAuth.ts

```diff
@@ -100,13 +100,15 @@ export const {
       },
     }),
   ],
-  secret: process.env.TOKEN_SECRET,
+  // When building the app use a random UUID as the token secret
+  secret: process.env.TOKEN_SECRET ?? crypto.randomUUID(),
```

Comment:
Would we need to point out that we used this crypto method (WebCrypto https://w3c.github.io/webcrypto/#Crypto-method-randomUUID) for our Protected B status?
I'm assuming this is fine but thought I'd ask just encase.

Reply:
No because the key is a throw away that is only used at build time.  It's only there to satisfy Next-Auths constructor.  Once the application 'starts' the real secret key that is passed in as an environment variable is used.